cloud/kubernetes/README.md

```diff
@@ -186,11 +221,16 @@ If you created a secure cockroach cluster, run:
 kubectl create -f example_app_secure.yaml
 ```
 
-For every pod being created, you will need to approve its client certificate request:
+When the first pod is being initialized, you will need to approve its client certificate request:
 ```shell
-kubectl certificate approve client.root-example-secure-etc..
+kubectl certificate approve client.root
 ```
 
+If more pods are then added through `kubectl scale deployment example-secure --replicas=X`, the generated
+certificate will be reused.
+**WARNING**: the example app in secure mode should be started with only one replica, or concurrent and
```

Comment:
I can definitely do 1. and 2. I think 3. should just cause any client to bail out, or we may be spamming with new CSRs when they've been intentionally denied. 4. can't work without storing the key of the original requester.